integrationos-api/src/endpoints/connection_oauth_definition.rs

```diff
@@ -116,6 +118,7 @@ impl RequestExt for CreateRequest {
             },
             frontend: Frontend {
```

Comment:
Why do you not create a method to initialize the Frontend Object? And avoid to have duplicated code ? It's more easily to maintain.

Line:119
Line:180